lib/RESTClient.js

```diff
@@ -393,6 +395,53 @@ class RESTClient {
             callback(errors.InternalError);
         }).end();
     }
+
+    /**
+    *   send a request to get all raft sessions
+    *   get server's response and return it
+    *
+    *   @param {string} reqUids - the identifier of the request
+    *   @param {callback} callback - callback(err, info)
+    *   @param {werelogs.Logger} [reqLogger] - Logger instance
+    *   @return {undefined}
+    */
+    getAllRafts(reqUids, callback, reqLogger) {
+        const log = reqLogger || this.createLogger(reqUids);
+        const path = '/_/raft_sessions/';
+        log.debug('getting all raftSessions');
+        this.request('GET', path, log, null, null, callback);
+    }
+
+    /**
+    *   Get raft logs from bucketd
+    *   get server's response and return it
+    *
+    *   @param {string} raftId - raft session id
+    *   @param {number} [start=1] - starting sequence number
+    *   @param {number} [end=10000] - ending sequence number
+    *   @param {boolean} [targetLeader=false] - true: from leader instead of
+    *       follower
+    *   @param {string} reqUids - the identifier of the request
+    *   @param {callback} callback - callback(err, info)
+    *   @param {werelogs.Logger} [reqLogger] - Logger instance
+    *   @return {undefined}
+    */
+    getRaftLog(raftId, start, end, targetLeader, reqUids, callback, reqLogger) {
```

Comment:
By `limit` parameter, you mean the `end` one; or do you mean that you'd rather use the semantic of `limit` (max nb of entries to get) rather than `end` (last sequence to retrieve) ?
Anyways, As I don't think it's a blocker for now, we'll try to go forward with this, and we can add minor adjustements afterwards; hopefully this'll be fine for you